build.gradle

```diff
@@ -18,7 +18,7 @@ import scripts.*
 ext {
   baseVersion = '3.0.0-SNAPSHOT'
   offheapVersion = '2.2.1'
-  managementVersion = '2.2.0'
+  managementVersion = '2.2.1'
```

Comment:
hum... that's an api change that is pending for terracotta-management ; I'd rather assume 2.3.0 will be the next version...